build/scripts/generate-aws-interfaces.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -ex
+
+[ "$#" -eq 2 ] || { echo "usage: ${0} <service-name> <interface-name>"; exit 1; }
+
+SERVICE_NAME="${1}"
+INTERFACE_NAME="${2}"
+
+PACKAGE_NAME="github.com/aws/aws-sdk-go-v2/service/${SERVICE_NAME}"
+AWS_SDK_DIR=$(go list -m -f '{{.Dir}}' "${PACKAGE_NAME}")
```

Comment:
Yeah, this is something we need to do manually before generating the interfaces because each service is now a Go module. A simple `go get <service-package>` before generating the interfaces will suffice.

Reply:
can this be written down somewhere? maybe next to the go generate lines so folks adding new interfaces know what to do

Reply:
I think that can just be part of the script, right? Do a go get ... `asg` beforehand? No need to do that manually.

Reply:
the name of the package differs sometimes, e.g. `asg` is `autoscaling` for the go mod. but it could be added

Reply:
Yup, I can add `go get` to the script.

Reply:
I have added a `go get` to the script.